src/styles/index.scss

```
@@ -1,3 +1,108 @@
+@import 'variables';
+@import 'star';
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+.card {
+  width: 200px;
+  height: 408px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__img-container {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  &__img {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__infos {
+    margin-top: 40px;
+  }
+
+  &__name {
+    font-family: $font;
+    color: $primaryColor;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    text-align: left;
+    margin: 0;
+  }
+
+  &__code {
+    font-family: $font;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondaryColor;
+    margin-top: 4px;
+    margin-bottom: 0;
+  }
+
+  &__reviews {
+    margin-top: 7px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  &__number {
+    font-size: 10px;
+    font-weight: 400;
+    font-family: $font;
+  }
+
+  &__price {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  &__text {
+    font-family: $font;
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: $secondaryColor;
+  }
+
+  &__value {
+    font-family: $font;
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $primaryColor;
+  }
+
+  &__button {
+    background-color: $blueAccent;
+    color: $whiteAccent;
+    border: none;
+    border-radius: 5px;
+    text-align: center;
+    text-decoration: none;
+    box-sizing: border-box;
+    width: 166px;
+    height: 40px;
```

don't hardcode height here as well
use proper padding instead
```suggestion
    width: 100%;
```